pkg/orchestrator/assetscanwatcher/watcher.go

```diff
@@ -330,10 +330,18 @@ func (w *Watcher) cleanupResources(ctx context.Context, assetScan *models.AssetS
 
 	switch w.scannerConfig.DeleteJobPolicy {
 	case DeleteJobPolicyNever:
-		assetScan.ResourceCleanup = utils.PointerTo(models.ResourceCleanupStateSkipped)
+		assetScan.ResourceCleanup = &models.ResourceCleanupStatus{
```

Comment:
Worth having a function here IMO
```
func createStatus(LastTransitionTime time.Time, message, reason string, state models.ResourceCleanupStatusState) *models.ResourceCleanupStatus {
	return &models.ResourceCleanupStatus{
		LastTransitionTime: utils.PointerTo(LastTransitionTime),
		Message:            utils.PointerTo(message),
		Reason:             utils.PointerTo(reason),
		State:              utils.PointerTo(state),
	}
}

```

Reply:
Agreed., but I would name the contructor as `NewResourceCleanupStatus(...)` to make it aligned with the naming convention Golang has.